scripts/generate_validation_data_xlinear.py

```diff
@@ -0,0 +1,72 @@
+""" Script to generate validation data from Wellcome Grants """
+import os
+
+import pandas as pd
+
+try:
+    from datascience.warehouse.warehouse import FortyTwo
+    from datascience.grants.cleaning import clean_grants
+except ImportError as e:
+    raise ImportError(f"Error importing {e.name }. "
+                      f"To use this script you need to install Wellcome's datascience internal "
+                      f"utils, for example with `make install-private-requirements`")
+
+
+# Name of the file output of the pipeline
+predictions_file = 'mesh_pipeline_result.csv'
+
+# Name of the file with the predicitons, alongside metadata
+merged_predictions_file = 'merged_mesh_predictions_mesh_xlinear_for_validation.xlsx'
+here = os.path.abspath(os.path.dirname(__file__))
+
+grants = pd.read_csv(os.path.join(here, '../data/raw/grants.csv'), index_col=0)
+grants = clean_grants(grants).fillna('')
+
+## Query FortyTwo to obtain the MeSH descriptors (could query the raw XML too?)
+descriptor_query = """
+SELECT [MeSH Lowest Level Classification], [MeSH Classification Description]
+FROM FortyTwo.dbo.MeSHClassificationHierarchy 
+"""
+
+forty_two = FortyTwo()
+descriptors = {
```

Comment:
so this script queries directly FortyTwo instead of using the results from the mesh pipeline, why?